ui/tests/unit/adapters/identity/entity-alias-test.js

```diff
@@ -6,37 +6,28 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
 import testCases from './_test-cases';
-import apiStub from 'vault/tests/helpers/noop-all-api-requests';
+import { setupMirage } from 'ember-cli-mirage/test-support';
 
 module('Unit | Adapter | identity/entity-alias', function (hooks) {
   setupTest(hooks);
-
-  hooks.beforeEach(function () {
-    this.server = apiStub();
-  });
-
-  hooks.afterEach(function () {
-    this.server.shutdown();
-  });
+  setupMirage(hooks);
 
   const cases = testCases('identity/entity-alias');
 
   cases.forEach((testCase) => {
     test(`entity-alias#${testCase.adapterMethod}`, function (assert) {
-      assert.expect(2);
+      assert.expect(3);
+      const method = testCase.method.toLowerCase();
+      const url = testCase.url.replace('/v1', '').split('?')[0];
+      const queryParams = testCase.url.includes('?list=true') ? { list: 'true' } : {};
+      this.server[method](url, (schema, req) => {
+        assert.ok(true, `${testCase.adapterMethod} calls the correct url with: ${testCase.url}`);
```

Comment:
The testcase url actually does not match the mirage url here, so it won't be a match. If we make it match, it's sort of not testing anything anyway. I can replace it with assert.true though -- the point is that it's hitting this server API path, so that is matching the test case